.github/workflows/build.yml

```diff
@@ -0,0 +1,349 @@
+name: Automated Test Build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Be aware of cryptocurrency-mining PR attacks. We may have to restrict to trusted members' PR only.

Reply:
It is a thing GitHub should care about.